src/ui/windows/TogglDesktop/TogglDesktop/ui/views/Timer.xaml.cs

```diff
@@ -27,64 +26,16 @@ public partial class Timer
         public Timer()
         {
             this.InitializeComponent();
-
-            this.setupSecondsTimer();
+            ViewModel = new TimerViewModel(false);
+            ViewModel.WhenValueChanged(x => x.IsRunning).Subscribe(_ => StartStopClick?.Invoke(this, EventArgs.Empty));
+            ViewModel.WhenValueChanged(x => x.DurationText).Subscribe(e => RunningTimeEntrySecondPulse?.Invoke(this, e));

```

Comment:
It's easy to change the subscribers of `RunningTimeEntrySecondPulse` to subscribe to the WhenValueChanged observable instead. This way there will be no need for the `RunningTimeEntrySecondPulse` event. Same applies to the `StartStopClick` event.